.github/actions/argus-builder/build-prep/src/main.ts

```diff
@@ -0,0 +1,259 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import { minimatch } from 'minimatch';
+// eslint-disable-next-line import/no-relative-packages
+import { findChangedFiles } from '../../../find-changed-files/src/findChangedFiles';
+// eslint-disable-next-line import/no-relative-packages
+import { validateJsonSchema } from '../../../validate-json-schema/src/validateJsonSchema';
+// eslint-disable-next-line import/no-relative-packages
+import { getCommaDelimitedArrayInput, ProcessedImage } from '../../common';
+
+// eslint-disable-next-line import/no-relative-packages
+export { ProcessedImage } from '../../common';
+
+type Inputs = {
+  githubToken: string
+  images: Record<string, any>
+  pathFilters: string[]
+  branchesInclude: string[]
+  branchesIgnore: string[]
+  manifestTriggerLabels: string[]
+};
+
+export function getInputs(): Inputs {
+  return {
+    githubToken: core.getInput('github_token', { required: true }),
+    images: JSON.parse(core.getInput('images', { required: true })),
+    pathFilters: getCommaDelimitedArrayInput('path_filters', { required: true }),
+    branchesInclude: getCommaDelimitedArrayInput('branches_include', { required: true }),
+    branchesIgnore: getCommaDelimitedArrayInput('branches_ignore', { required: false }),
+    manifestTriggerLabels: getCommaDelimitedArrayInput('manifest_trigger_labels', { required: true }),
+  };
+}
+
+const imagesInputSchema = {
```

Comment:
this is new right? I think we'll need to document this somewhere with examples

Reply:
Sort of, it was formerly in `argus-docker-build.yaml`: https://github.com/chanzuckerberg/github-actions/blob/85706b317c52de7c9b4b7249d15d81f41bc5ac2e/.github/workflows/argus-docker-build.yaml#L101-L145